### PR TITLE
Moving cloud accounts storage to the etc, storage layer.

### PR DESCRIPTION
**Pull request by yegorch**
This is the first step on gradually moving data layer to the etcd.

### CODE REVIEW

File: pkg/util/test_asserts.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	cl, err := client.New(client.Config{
+		Endpoints: []string{etcdHost},
+	})
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	defer cancel()
+	vers, err := cl.GetVersion(ctx)
+	fmt.Println(vers)
```
Comment (stgleb) [inline pkg/util/test_asserts.go]: I guess this print is redundant?

File: pkg/cloud_account/storage.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		Recursive: true,
+	})
+	if err != nil {
+		if clientErr, ok := err.(client.Error); ok {
+			//If account is not found return nil
+			if clientErr.Code == client.ErrorCodeKeyNotFound {
+				return accounts, nil
+			}
+		}
+		return accounts, errors.WithStack(err)
```
Comment (saheienko) [inline pkg/cloud_account/storage.go]: Why do you use WithStack here?

File: pkg/cloud_account/storage.go [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+	return accounts, nil
+}
+
+//Create stores account in etcd, if account with such name is already present it will return an error
+func (r *ETCDRepository) Create(ctx context.Context, acc *CloudAccount) (error) {
+	rawJson, err := json.Marshal(acc)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	_, err = r.keysAPI.Create(ctx, prefix+acc.Name, string(rawJson))
```
Comment (saheienko) [inline pkg/cloud_account/storage.go]: Is it better to use getPath func here? (instead of prefix+acc.Name)

File: pkg/cloud_account/storage.go [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+		for _, v := range resp.Node.Nodes {
+			ca := CloudAccount{}
+			err = json.NewDecoder(strings.NewReader(v.Value)).Decode(&ca)
+			if err != nil {
+				logrus.Warningf("corrupted data in etcd node %s", v.Key)
+				continue
+			}
+			accounts = append(accounts, ca)
+		}
+	}
```
Comment (saheienko) [inline pkg/cloud_account/storage.go]: Could you wrap this block with getCloudAccounts?

File: pkg/cloud_account/storage.go [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+	resp, err := r.keysAPI.Get(ctx, prefix, &client.GetOptions{
+		Recursive: true,
+	})
+	if err != nil {
+		if client.IsKeyNotFound(err) {
+			return accounts, nil
+		}
+		return accounts, errors.WithStack(err)
+	}
+	if resp.Node != nil {
```
Comment (saheienko) [inline pkg/cloud_account/storage.go]: Could you wrap this block with getCloudAccounts?

File: pkg/cloud_account/storage_integration_test.go [hunk 6]
```diff
@@ -0,0 +1,78 @@
+//+build integration
```
[the remark this answers is not in the document]
Comment (yegorch) [reply pkg/cloud_account/storage_integration_test.go]: Let's do it after the integration testing infrastructure story is ready.

**Pull request merged**